identity_presentation/src/commonMain/kotlin/net/thechance/mena/identity/presentation/screen/uploadProfileImage/components/UploadImageContainer.kt

```diff
@@ -0,0 +1,152 @@
+package net.thechance.mena.identity.presentation.screen.uploadProfileImage.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import mena.identity_presentation.generated.resources.Res
+import mena.identity_presentation.generated.resources.click_to_upload
+import mena.identity_presentation.generated.resources.ic_add_image
+import mena.identity_presentation.generated.resources.pencil_edit
+import net.thechance.mena.designsystem.presentation.component.icon.Icon
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.MenaTheme
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import net.thechance.mena.identity.presentation.utils.dashedBorder
+import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.compose.ui.tooling.preview.Preview
+
+@Composable
+fun UploadImageContainer(
+    onClick: () -> Unit,
+    image: ImageBitmap?,
+    modifier: Modifier = Modifier,
+) {
+    val borderColor = Theme.colorScheme.brand.brand
+    val radius = Theme.radius.xl
+
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(bottom = 20.dp)
+    ) {
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .aspectRatio(1f / 1f)
```

Comment:
No need for division here:
```suggestion
                .aspectRatio(1f)
```